### ISSUE TITLE
Make some process videos

### ISSUE THREAD

**Issue by JackiePease** [DoESLiverpool/covid19#82]
Gina Czarnecki has offered to make some videos of our production processes so we don't have so much to explain to each new volunteer.
She is coming in at 9:15 today and @dazz-ling has volunteered to show her the processes.

**Comment by JackiePease** [comment]
Jane Dillon has made 2 videos showing the cleaning process and has emailed them to me. I will upload them to here at lunchtime.

**Comment by JackiePease** [comment]
Tried to upload Jane's videos to repository, might not have been best idea. Git recommending something called large file storage. Does anyone know anything about that?

**Comment by dfaligertwood** [comment]
Probably easier to just put them on google drive or similar and link them from production.md if you haven't already worked out git LFS. IIRC you need to be using an offline client (github client or git-lfs on the command line) to use it.

**Comment by amcewen** [comment]
Agreed @dfaligertwood, we've got DoES YouTube and Vimeo channels, so sticking them up there would seem to make most sense, and then link from `production.md`.

It'd be good to get Gina (or someone) to film a "how to assemble a visor" guide, for us to add as a link to the packing note.

**Comment by afroleft** [comment]
> Gina Czarnecki has offered to make some videos of our production processes so we don't have so much to explain to each new volunteer.

Brilliant. If there's anything else we can do through website, video, checklists to support onboarding, let us know @JackiePease X
